docs/developer-guide/get-started/deployment/emt-virt-mf.md

```diff
@@ -0,0 +1,1104 @@
+
+The default launch command without any parameters is for 2 cores and 2G RAM. You can change that with startup parameters:
+
+Example guest start configuration for 4 cores, 4G RAM:
+
+```bash
+# Add -m option to specify 4G of memory
+# Add -c option to specify 4 cpu cores for guest VM
+sudo -E ./start_windows.sh -m 4G -c 4
+```
+
+##### Enabling USB Devices in Windows Guest VM**
+
+For Windows guest VMs, USB devices can be setup in two ways:
+
+1. Passthrough of all USB Host Devices
+
+USB host passthrough parameter option can be added in the launch command to passthrough all USB devices on the USB host.
+
+Add an additional parameter to the Guest VM launch command as shown below.
+
+```bash
+# Note: all connected USB devices will be passthrough to the
+guest VM with USB host passthrough option
+sudo -E ./start_windows.sh --passthrough-pci-usb
+```
+
+2. Passthrough of Selected USB Devices
+
+An external command option can be used to passthrough only a few selected USB devices.
+
+Retrieve the vendorid and productid of USB device. In this example, `046d` is vendor ID, `c06a` is product ID.
+
+```bash
+# On target terminal.
+lsusb
+Bus 004 Device 003: ID 046d:c06a Logitech, Inc. USB Optical
+Mouse
+```
+
+Add an additonal parameter to the Guest VM launch command:
+
+```bash
+# Add extra command when start guest
+sudo -E ./start_windows.sh -e "-device usb￾host,vendorid=0x046d,productid=0xc06a"
+```
+
+> **Note:** A passthrough device option can only be used once because a device can be passthrough to only 1 guest VM at a time.
+
+##### Enabling PCIe Wi-Fi Adapter Device in Windows Guest VM**
+
+For Windows guest VMs, PCI Wi-Fi device passthrough can be setup by adding `--passthrough-pci-wifi` parameter to guest VM launch command:
+
+```bash
+# Add --passthrough-pci-wifi for passing through Wifi adapter
+sudo -E ./start_windows.sh --passthrough-pci-wifi
+```
+
+## Ubuntu
+
+The sections below describe setup, running and configuration of Ubuntu VM using EMT Virt.
+
+### Setup Ubuntu VM
+
+#### Ubuntu VM Setup Prerequisites
+
+##### Guest Operating System Requirements
+
+Download [Ubuntu 24.04.2 LTS](https://releases.ubuntu.com/noble/ubuntu-24.04.2-desktop-amd64.iso) from the official Ubuntu website.
+
+<!-- old ver unavailable, link updated, verify version compatibility -->
+
+##### Guest Kernel Files Required
+
+Obtain the `*.deb` kernel files that have been used for setting up the host.
+
+##### Ubuntu Guest VM -  Required Installation Scripts
+
+- `sriov_patches.zip`
+- `ubuntu_kvm_multios_scripts.zip`
+
+Obtain the scripts from the release package [RPL-S_RPL-SR_KVM_MultiOS.zip](https://www.intel.com/content/www/us/en/secure/design/confidential/software-kits/kit-details.html?kitId=839117)
+
+##### Make a Separate Copy of OVMF for Ubuntu
+
+1. Create a separate copy of OVMF for Ubuntu VM use:
+
+```bash
+# Make a copy of OVMF for Ubuntu guest
+ln -sf ovmf/OVMF_CODE.fd OVMF_CODE.fd
+cp ovmf/OVMF_VARS.fd OVMF_VARS_ubuntu.fd
+```
```

Comment:
````suggestion
   ```bash
   # Make a copy of OVMF for Ubuntu guest
   ln -sf ovmf/OVMF_CODE.fd OVMF_CODE.fd
   cp ovmf/OVMF_VARS.fd OVMF_VARS_ubuntu.fd
   ```
````